source/utilities.tex

```diff
@@ -10571,6 +10580,11 @@
   template<class R, class... ArgTypes>
     class move_only_function<R(ArgTypes...) @\cv{}@ @\placeholder{ref}@ noexcept(@\placeholder{noex}@)>; // \seebelow
 
+  // \ref{func.wrap.ref}, non-owning wrapper
+  template<class... S> class function_ref;                          // \notdef, freestanding
+  template<class R, class... ArgTypes>
+    class function_ref<R(ArgTypes...) @\cv{}@ noexcept(@\placeholder{noex}@)>;           // \seebelow, freestanding
```

Comment:
I thought it was better to have the "freestanding" comments be last - that way the `\undef`/`\seebelow` comments better line up more with the ones just above it, and the `freestanding` comments also better line up. Thoughts anyone? Please chime in if you have an opinion, else we can leave this to The Editor :)

Reply:
I don't have a strong opinion, and this is the first time this situation comes up. I'll go with the ordering from the paper, because I think that "// freestanding" is an important global marker that is worth taking a consistent and predictable form, even if there are competing interests here. We can revisit this later if more such cases come up.